imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -81,16 +86,15 @@ class CaseExplorer extends Component {
     (a, b) => a.length === b.length
   )
   makeCaseGrouping = memoizeOne(
-    (caseList, showOpen, onlyAssigned, allNotifs, unreadNotifs) => {
-      const openFilter = showOpen ? x => !isClosed(x) : x => isClosed(x)
-      const assignedFilter = onlyAssigned ? x => x.assignee === this.props.currentUser.bugzillaCreds.login : x => true
-
+    (caseList, roleFilterValues, sortBy, allNotifs, unreadNotifs) => {
+      const assignedFilter = roleFilterValues === 'All' ? x => true
```

Comment:
2 levels of nested ternary logic make this extremely hard to read. Should be broken down to separate statements